controllers/virtualmachinepublishrequest/virtualmachinepublishrequest_controller.go

```diff
@@ -54,7 +55,7 @@ const (
 
 var (
 	// DefaultRequeueDelaySeconds represents the default requeue delay seconds when reconcile.
-	DefaultRequeueDelaySeconds = 60
+	DefaultRequeueDelaySeconds int64 = 60
```

Comment:
Never, ever, *ever* use a package level variable as anything but a constant. This is nothing but potential for race conditions. Rewrite this as a constant and then provide the ability to inject a new requeue some other way.